### ISSUE TITLE
Adding an impeded label to an issue when marked as blocked in HuBoard

### ISSUE THREAD

**Issue by patdolem** [huboard/huboard#399]
We have scripts using an impeded label to show progress on charts. Some people are also viewing issues status directly in Github. 

[DUPLICATE] #659

**Comment by droopy12** [comment]
The same would also be nice for "Ready for next stage"

**Comment by igorsantos07** [comment]
That makes sense. I actually never noticed there was this option on Huboard. Instead, I've implemented that by having an additional "On Hold" column on my board. That helps me as it frees other columns of tasks I can't work on, while I can still take a look at them once in a while to see if there are updates.

**Comment by chrowe** [comment]
Another advantage of doing this would be that you could filter by "Ready for next stage" or "On Hold" in Huboard just using labels

**Comment by discorick** [comment]
[Blocked] There are some modeling constraints and css view changes that need to be addressed before this can be completed.

**Comment by benjamincharity** [comment]
(moving comment here from duplicate issue)

One of the major benefits of Github issues (imo) is the historical timeline. eg `user_1 added the 2 - Needs Discussion label 17 hours ago`. Because the Huboard `Mark as blocked` option is so available, it's hard for my team members to not use it. But if they _do_ use and remove it there is no way to know the label ever existed. When was it unblocked? Who unblocked it?

---

- Create ticket
- Mark ticket as blocked within Huboard
- Unmark ticket as blocked.
- No way to know if it was ever blocked and/or who blocked/unblocked it.

**Comment by discorick** [comment]
Issues that are blocked/ready now apply the "blocked/ready" label if it is present, would you consider this issue resolved @patdolem @benjamincharity ?

I know theres been some talk about being able to customize which label maps to the blocked/ready state, I would be inclined to open a new issue for that.

**Comment by benjamincharity** [comment]
> able to customize which label maps to the blocked/ready state

I know at least for my case, the new blocked/ready labels don't solve anything unless I want to go through all of my companies repos and changed the label names to match what Huboard defines rather than the existing format across other labels. So for me it is not a workable solution.

> I would be inclined to open a new issue for that.

I have no problem with that. A cross link would be nice in case others come across this issue. 👍

**Comment by patdolem** [comment]
> able to customize which label maps to the blocked/ready state

I have the same problem as for @benjamincharity. We have a huge number of repos we would have to go through to change the label name. Also our team members could be confused with a new name.

> I would be inclined to open a new issue for that.

What does it mean in term of release schedule?

**Comment by dahlbyk** [comment]
> @patdolem: What does it mean in term of release schedule?

No specific plans at this point. Let's continue the discussion over in https://github.com/huboard/huboard/issues/695.
